src/pages/program/[slug]/[profileRoute]/index.tsx

```diff
@@ -153,7 +153,7 @@ const ReadOnlyHome = ({
     //TODO: Figure out whether the buttons at the top should be sticky
     <div className="box-border bg-tertiary min-h-full pt-16 lg:pt-32">
       {inProgram ? (
-        <LinkToProgram className="transform -translate-y-20" />
```

Comment:
Why are we doing this? I think it could still be helpful?

Reply:
I think we need to talk about this then- Noah had wanted it to be admin-shareable only, but there may be different use cases where the link could be useful to mentors/mentees.

Reply:
I asked noah and he's cool with leaving it